draft-ietf-rats-corim.md

```diff
@@ -1100,24 +1100,88 @@ The `uint` and `text` types MUST NOT be interpreted in a global scope.
 
 #### Reference Values Triple {#sec-comid-triple-refval}
 
-A Reference Values triple relates reference measurements to a Target
-Environment. For Reference Value Claims, the subject identifies a Target
-Environment, the object contains measurements, and the predicate asserts that
-these are the expected (i.e., reference) measurements for the Target
-Environment.
+The Reference Values Triple has the following structure:
 
 ~~~ cddl
 {::include cddl/reference-triple-record.cddl}
 ~~~
 
+The `reference-triple-record` has the following parameters:
+
+* `ref-env`: Search criterion that locates an Evidence environment that matches the reference environment.
+* `ref-claims`: Search criteria that locates the Evidence measurements that match the reference Claims.
+
+To process `reference-triple-record` both the `ref-env` and `ref-claims` criteria are compared with Evidence entries.
+If the search criteria are satisfied, the matching entry is re-asserted, except with the Reference Value Provider's authority.
+By re-asserting Evidence using the RVP's authority, the Verifier can avoid mixing Reference Values (reference state) with Evidence (actual state).
+See {{-rats-endorsements}}.
+Re-asserted Evidence using RVP authority is said to be "corroborated".
+
 #### Endorsed Values Triple {#sec-comid-triple-endval}
 
-An Endorsed Values triple declares additional measurements to add to the ACS.
+The Endorsed Values Triple has the following structure:
 
 ~~~ cddl
 {::include cddl/endorsed-triple-record.cddl}
 ~~~
 
+The `endorsed-triple-record` has the following parameters:
+
+* `condition`: Search criterion that locates Evidence, corroborated Evidence, or Endorsements.
```

Comment:
envidence [etc] _environment_, if we want to keep this consistent with RVT above

Reply:
> envidence [etc] _environment_, if we want to keep this consistent with RVT above

Addressed by commit https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/299/commits/31bc933253de02567f3c9dc184771be189436f6e